ScrapingToolKit/README.md

```diff
@@ -7,6 +7,9 @@ STK is installed by downloading the files to your local machine.  This can eithe
 
 Dependencies:
 `pip install lxml`
```

Comment:
for these we could tell people to run:

```
pip install -r requirements.txt
```

Reply:
updated readme to tell people to install dependencies via requirements.txt